src/Search/FluentSolrIndex.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace TractorCow\Fluent\Search\Solr;
+
+use SilverStripe\FullTextSearch\Solr\SolrIndex;
+use SilverStripe\CMS\Model\SiteTree;
+
+/**
+ * Default search index
+ */
+class FluentSolrIndex extends SolrIndex
```

Comment:
We don't really need this, since it is essentially a standard Solr index and is covered in the FTS documentation already